### PR TITLE
Track media properties on upload

### PR DESCRIPTION
**Pull request by daniloercoli**
Add media properties to `photo_added` and `video_added` Analytics.

Ref: #5168 (do not close the ticket since I still want to be sure we're tracking what's really needed).

cc @mzorz

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java [hunk 1]
```diff
@@ -1542,13 +1542,44 @@ public void onFailure(ApiHelper.ErrorType errorType, String errorMessage, Throwa
...
 
-    private void trackAddMediaEvents(boolean isVideo, boolean fromMediaLibrary) {
+    /**
+     * Analytics about new media
+     *
+     * @param isVideo Whether is a video or not
+     * @param uri The URI of the media on the device, or null
+     * @param path The path of the media on the device, or null
+     */
+    private void trackAddMediaFromDevice(boolean isVideo, Uri uri, String path) {
```
Comment (mzorz) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java]: I think renaming this to `trackAddMediaFromDeviceEvents` would be more descriptive

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java [hunk 2]
```diff
@@ -1542,13 +1542,44 @@ public void onFailure(ApiHelper.ErrorType errorType, String errorMessage, Throwa
...
+        }
+    }
+
+    /**
+     * Analytics about media already available in the blog's library.
+     *
+     * @param isVideo Whether is a video or not
+     * @param mediaId The ID of the media in the WP blog's library, or null if device media.
+     */
+    private void trackAddMediaFromWPLibrary(boolean isVideo, String mediaId) {
```
Comment (mzorz) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java]: Same as above, could be better named with `trackAddMediaFromWPLibraryEvents`

**Comment by daniloercoli** [comment]
Ready for another review!

**Comment by mzorz** [comment]
:shipit:

**Pull request merged**